addons/website/controllers/main.py

```diff
@@ -975,3 +976,11 @@ def website_content_image(self, id=None, max_width=0, max_height=0, **kw):
             if unique:
                 kw['unique'] = unique
         return self.content_image(**kw)
+
+
+class WebsiteSession(Session):
+
+    # Force auth='public', required for logout
+    @http.route(auth="public")
+    def logout(self, *args, **kw):
+        return super().logout(*args, **kw)
```

Comment:
- Could you move this above the Class just before as it is about compatibility routes?
- Maybe we should put `website=True` at the same time? Why not :shrug:

Reply:
@madprog Forgot one question: why only in 17.2? Was there something in there that created the bug? :pray: